cellprofiler/modules/identifyobjectsmanually.py

```diff
@@ -20,16 +30,6 @@
 -  *Zoom out:* Reverses the effect of the last zoom-in.
 -  *Erase:* Erases an object if you click on it.
 
-|
```

Comment:
Please restore this table and the ending `"""` which close out this documentation string (L23-L32).